code/game/response_team.dm

```diff
@@ -129,29 +95,50 @@ var/ert_request_answered = 0
 		if(index > emergencyresponseteamspawn.len)
 			index = 1
 
-		var/client/C = M.client
-		var/mob/living/carbon/human/new_commando = C.create_response_team(emergencyresponseteamspawn[index])
-		new_commando.mind.key = M.key
-		new_commando.key = M.key
-		new_commando.update_icons()
+		spawn(0)
```

Comment:
Goal: one ERT member being AFK during spawn shouldn't block the rest of the squad from spawning. Would appreciate close review of the logic on this one.